AirCasting/Graph/Graph.swift

```diff
@@ -64,6 +60,30 @@ struct Graph: UIViewRepresentable {
         let allLimitLines = getLimitLines()
         uiView.limitLines = allLimitLines
         simplifyGraphline(entries: entries, uiView: uiView)
+        context.coordinator.numberOfMeasurements = stream.allMeasurements?.count ?? 0
+        context.coordinator.currentThreshold = ThresholdWitness(sensorThreshold: thresholds)
+        return uiView
+    }
+    
+    func updateUIView(_ uiView: AirCastingGraph, context: Context) {
+        let thresholdWitness = ThresholdWitness(sensorThreshold: self.thresholds)
+        
+        if context.coordinator.currentThreshold != thresholdWitness ||
```

Comment:
guard would save us 1 indentation level here

Reply:
Why not then, changing!